storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2192,7 +2192,7 @@ void get_id_from_mysql_table_scan(const std::string name, uint64 *id,
 
       dd_process_schema_rec(heap, rec, sys_tables, &mtr, &rec_name,
                             &rec_schema_id);
-      mysql_schema.insert(std::pair<std::string, int>(rec_name, rec_schema_id));
+      mysql_schema.insert(std::make_pair(rec_name, rec_schema_id));
```

Comment:
when do we free this?

Reply:
fixed.

Reply:
we free it inside dict_load_from_spaces_sdi